netbox/dcim/models/cables.py

```diff
@@ -501,9 +498,15 @@ def from_origin(cls, terminations):
             # Terminations must all be of the same type
             assert all(isinstance(t, type(terminations[0])) for t in terminations[1:])
 
+            # All mid-span terminations must all be attached to the same device
```

Comment:
Is this required by the current implementation ?

Reply:
I am doing it to maintain the current status quo and not introduce any new features (beyond those requested in the Bug)